amethyst_core/src/deferred_dispatcher_operation.rs

```diff
@@ -1,178 +0,0 @@
-//! Provides the ability to store `Systems`, `Bundles`, `Barriers`, in a normal vector for deferred dispatcher construction.
```

Comment:
Why is this functionality being deleted?

Reply:
This was only needed for specs to lazily build systems. Legion system builds don't have any dependencies on `World` or `Resources` and can be initialized right away. Similar functionality is now provided by system bundles.